### PR TITLE
ARTEMIS-1774 Node Manager Store table name should be configurable

### PR DESCRIPTION
**Pull request by franz1981**
It exposes the table name configuration for:
- CLI command
- configuration file

The docs are updated as well.

### CODE REVIEW

**Comment by franz1981** [comment]
@clebertsuconic I have added CLI `Create` and `DBOption` configuration too.
The first because it was already handling the other JDBC HA configuration options, while the second one because it seems to deal with each JDBC table name configuration, but probably it is not necessary.
Please let me know if I need change/drop any of them, because unecessary :+1:

File: artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/Create.java [hunk 1]
```diff
@@ -278,9 +278,12 @@
    @Option(name = "--jdbc-large-message-table-name", description = "Name of the large messages table")
    private String jdbcLargeMessages = ActiveMQDefaultConfiguration.getDefaultLargeMessagesTableName();
 
-   @Option(name = "--jdbc-page-store-table-name", description = "Name of the page sotre messages table")
+   @Option(name = "--jdbc-page-store-table-name", description = "Name of the page store messages table")
```
Comment (clebertsuconic) [inline artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/Create.java]: :)

File: artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/tools/DBOption.java [hunk 2]
```diff
@@ -74,9 +74,12 @@
    @Option(name = "--jdbc-large-message-table-name", description = "Name of the large messages table")
    private String jdbcLargeMessages = ActiveMQDefaultConfiguration.getDefaultLargeMessagesTableName();
 
-   @Option(name = "--jdbc-page-store-table-name", description = "Name of the page sotre messages table")
+   @Option(name = "--jdbc-page-store-table-name", description = "Name of the page store messages table")
    private String jdbcPageStore = ActiveMQDefaultConfiguration.getDefaultPageStoreTableName();
```
Comment (clebertsuconic) [inline artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/tools/DBOption.java]: There's something wrong here...

the default should be what's on the XML.

Comment (clebertsuconic) [reply artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/tools/DBOption.java]: nm... I thought you missed it on parseDBConfiguration

**Comment by clebertsuconic** [comment]
it looks good. will wait travis CI and merge it.

**Pull request closed without merging**